bin/bot.py

```diff
@@ -39,7 +39,8 @@ def get_secret(secret):
     'everyBot.cogs.animals.animals',
     'everyBot.cogs.random_compliment.random_compliment',
     'everyBot.cogs.dnd.dnd',
-    'everyBot.cogs.template.template'
+    'everyBot.cogs.template.template',

```

Comment:
i think we should take this out tbh

Reply:
Doesn't really have anything to do with this PR though? 😕

Reply:
That's true